app/src/main/java/ch/epfl/sdp/blindly/ui/main/SectionsPagerAdapter.kt

```diff
@@ -7,27 +7,35 @@ import androidx.fragment.app.FragmentPagerAdapter
 import ch.epfl.sdp.blindly.*
 
 private val TAB_TITLES = arrayOf(
-    R.string.main_page,
-    R.string.match_page,
-    R.string.message_page,
-    R.string.profile_page
+        R.string.main_page,
+        R.string.match_page,
+        R.string.message_page,
+        R.string.profile_page
 )
 
 /**
  * A [FragmentPagerAdapter] that returns a fragment corresponding to
  * one of the sections/tabs/pages.
  */
 class SectionsPagerAdapter(private val context: Context, fm: FragmentManager) :
-    FragmentPagerAdapter(fm) {
+        FragmentPagerAdapter(fm) {
 
     override fun getItem(position: Int): Fragment {
         // getItem is called to instantiate the fragment for the given page.
         // Return a PlaceholderFragment (defined as a static inner class below).
         return when (position) {
-            0 -> {MainPage()}
-            1 -> {MatchPage()}
-            2 -> {MessagePage()}
-            else -> {ProfilePage()}
+            0 -> {
+                MainPage()
+            }
+            1 -> {
+                MatchPage()
+            }
+            2 -> {
+                MessagePage()
+            }
+            else -> {
+                ProfilePage()
+            }
         }
     }

```

Comment:
Isn't it cleaner to have one-liners instead ? I'm not sure but in that case I think it's easier to read if we keep 0 -> { MainPage() }...

Reply:
Again they are all formatted automatically with our project's predefined indentation rules, if we want something else than we should change it rather than changing for each file manually which can lead to more errors.